server/src/main/resources/application.conf

```diff
@@ -86,6 +86,11 @@ email {
   senderAddress = ${?EMAIL_SENDER_ADDRESS}
 }
 
+aws {
+  region = "us-west-2"
+  region = ${?AWS_REGION}
```

Comment:
Please include the access key and the secret key, I prefer to have everything in a single place